api/pb/javascript/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@textile/threads-client-grpc",
-  "version": "0.0.2",
+  "version": "0.0.0",
```

Comment:
Remembering what we did for JS packages in go-textile... this will become the tag number in CI before a publish happens. Otherwise, this would still have to be updated manually.